itests/fe/group_join_test.py

```diff
@@ -11,6 +11,31 @@
     from tests.setup import SetupTest
 
 
+def test_request_to_join_group(tmpdir, setup, browser):
```

Comment:
How about instead moving the tests from the this file all into `group_view_test`? Having e.g. `test_request_to_join_group` and `test_remove_member` be in different files feels not great

Reply:
Hm, it feels correct to me. This is all of the tests that exercise the group join form, which `test_remove_member` does not do (it only exercises the group view page).